### ISSUE TITLE
Contribute the rest of the charts to the `visual-vocabulary`

### ISSUE THREAD

**Issue by huong-li-nguyen** [mckinsey/vizro#762]
## Thank you for contributing to our visual-vocabulary! 🎨 

Our visual-vocabulary is a dashboard, that serves a a comprehensive guide for selecting and creating various types of charts. It helps you decide when to use each chart type, and offers sample Python code using [Plotly](https://plotly.com/python/), and instructions for embedding these charts into a [Vizro](https://github.com/mckinsey/vizro) dashboard.

Take a look at the dashboard here: https://huggingface.co/spaces/vizro/demo-visual-vocabulary

## Instructions

0. Get familiar with the dev set-up (this should be done already as part of the initial intro sessions)

1. Read through the [README](https://github.com/mckinsey/vizro/tree/main/vizro-core/examples/visual-vocabulary) of the visual vocabulary

2. Follow the steps to contribute a chart. Take a look at other examples. This [commit](https://github.com/mckinsey/vizro/pull/634/commits/417efffded2285e6cfcafac5d780834e0bdcc625) might be helpful as a reference to see which changes are required to add a chart.

3. Ensure the app is running without any issues via `hatch run example visual-vocabulary`

4. List out the resources you've used in the [README](https://github.com/mckinsey/vizro/tree/main/vizro-core/examples/visual-vocabulary)

5. Raise a PR 

## EASY

- [ ] Cumulative curve 
- [x] Diverging bar
- [x] Diverging stacked bar
- [x] Dot plot
- [x] Gantt
- [x] Lollipop
- [x] Radar
- [x] Waterfall

## INTERMEDIATE / ADVANCED

**Recommended if you already have Plotly experience**

- [ ] Grid plot
- [ ] Barcode
- [ ] Bubble timeline
- [ ] Correlation matrix
- [ ] Slope
- [ ] Bullet
- [x] Pictogram
- [x] Bump
- [ ] Fan
- [ ] Network
- [ ] Radial
- [x] Sparkline
- [ ] Ordered bubble
- [ ] Surplus deficit
- [ ] Arc
- [ ] Chord
- [ ] Beeswarm
- [ ] Flow map
- [ ] Marimeko
- [ ] Venn

**Comment by MiD200998** [comment]
Hi Li, can i tackle Beeswarm? Sounds interesting to work on!

**Comment by huong-li-nguyen** [comment]
Yes @MiD200998 - you can try that! :) I'll create a separate ticket for you, but in the mean time feel free to start.

**Comment by ataraexia** [comment]
@huong-li-nguyen - Can I try sparkline?

**Comment by huong-li-nguyen** [comment]
Yes, @ataraexia - I'll create a separate ticket for you, but feel free to start 👍

**Comment by sanketjainindian** [comment]
can i try "Lollipop"?

**Comment by huong-li-nguyen** [comment]
Hey @sanketjainindian  the lollipop chart has already been taken and is in development in one of the open PRs already :)

**Comment by sanketjainindian** [comment]
@huong-li-nguyen Open to any if available for to create

**Comment by datascienceio** [comment]
Hi, I've submitted a pull request for adding Correlation Matrix.
